### PR TITLE
Add missing inline commands CLI setup

### PR DESCRIPTION
**Pull request by MadhukaHarith92**

### CODE REVIEW

File: components/cli/cmd/cellery/init.go [hunk 1]
```diff
@@ -25,12 +25,18 @@ import (
 )
 
 func newInitCommand() *cobra.Command {
+	var projectName = ""
 	cmd := &cobra.Command{
 		Use:   "init",
 		Short: "Initialize a cell project",
-		Args:  cobra.NoArgs,
+		Args: func(cmd *cobra.Command, args []string) error {
```
Comment (nadundesilva) [inline components/cli/cmd/cellery/init.go]: This callback should be only used for validating arguments and returning a proper error. This should check whether the number of arguments is 0 or 1 and return an error otherwise.

You can move the part where you actually get the arguments to the Run function.

Please check other commands and fix in other places as well

File: components/cli/cmd/cellery/init.go [hunk 2]
```diff
@@ -25,12 +25,18 @@ import (
 )
 
 func newInitCommand() *cobra.Command {
+	var projectName = ""
 	cmd := &cobra.Command{
 		Use:   "init",
```
Comment (nadundesilva) [inline components/cli/cmd/cellery/init.go]: Please update the usages to have the optional project argument

File: components/cli/cmd/cellery/init.go (hunk 1 above)
Comment (MadhukaHarith92) [reply components/cli/cmd/cellery/init.go]: Updated.

File: components/cli/cmd/cellery/init.go (hunk 2 above)
Comment (MadhukaHarith92) [reply components/cli/cmd/cellery/init.go]: Updated.

File: components/cli/cmd/cellery/setup_cleanup_existing.go [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+	var ingress = false
+	cmd := &cobra.Command{
+		Use:   "existing",
+		Short: "Cleanup existing cluster setup",
+		Args:  cobra.NoArgs,
+		Run: func(cmd *cobra.Command, args []string) {
+			commands.RunCleanupExisting(istio, ingress)
+		},
+	}
+	cmd.Flags().BoolVarP(&istio, "istio", "i", false, "Remove istio")
```
Comment (Mirage20) [inline components/cli/cmd/cellery/setup_cleanup_existing.go]: Shall we remove the short flags? It is much readable with long flags.

File: components/cli/cmd/cellery/setup_cleanup_gcp.go [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+package main
+
+import (
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cellery-io/sdk/components/cli/pkg/constants"
+
```
Comment (Mirage20) [inline components/cli/cmd/cellery/setup_cleanup_gcp.go]: Unnecessary new line. Please run `make code.format` to format the code.

File: components/cli/cmd/cellery/setup_cleanup_gcp.go [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+func newSetupCleanupGcpCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "gcp",
+		Short: "Cleanup gcp setup",
+		Args: func(cmd *cobra.Command, args []string) error {
+			err := cobra.ExactArgs(1)(cmd, args)
+			if err != nil {
+				return err
+			}
+			return commands.ValidateGcpCluster(args[0])
```
Comment (Mirage20) [inline components/cli/cmd/cellery/setup_cleanup_gcp.go]: Is this related to argument validation?

File: components/cli/cmd/cellery/setup_cleanup_gcp.go [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+		},
+		Run: func(cmd *cobra.Command, args []string) {
+			uniqueNumber := strings.Split(args[0], constants.GCP_CLUSTER_NAME)[1]
```
Comment (Mirage20) [inline components/cli/cmd/cellery/setup_cleanup_gcp.go]: Shall we check the slice length return from the Split function? Otherwise using index 1 directly may cause "slice bounds out of range" error

File: components/cli/cmd/cellery/setup_cleanup_local.go [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+	"github.com/cellery-io/sdk/components/cli/pkg/commands"
+)
+
+func newSetupCleanupLocalCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "local",
+		Short: "Cleanup local setup",
+		Args: func(cmd *cobra.Command, args []string) error {
+			if !commands.IsVmInstalled() {
```
Comment (Mirage20) [inline components/cli/cmd/cellery/setup_cleanup_local.go]: This is not an arg validation. Shall we move this to Run function?

File: components/cli/cmd/cellery/setup_create_existing_cluster.go [hunk 8]
```diff
@@ -19,25 +19,107 @@
...
+	var nfsServerIp = ""
+	var fileShare = ""
+	var dbHostName = ""
+	var dbUserName = ""
+	var dbPassword = ""
 	cmd := &cobra.Command{
 		Use:   "existing",
 		Short: "Create a Cellery runtime in existing cluster",
 		Args: func(cmd *cobra.Command, args []string) error {
+			if isPersistentVolume && hasNfsStorage {
```
Comment (Mirage20) [inline components/cli/cmd/cellery/setup_create_existing_cluster.go]: Same. Args is meant for validating arguments. You can use PreRunE for validating user input

File: components/cli/pkg/commands/setup_switch.go [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/cellery-io/sdk/components/cli/pkg/constants"
+	"github.com/cellery-io/sdk/components/cli/pkg/util"
+)
+
+func RunSwitchCommand(context string) error {
+	cmd := exec.Command(constants.KUBECTL, "config", "use-context", context)
```
Comment (Mirage20) [inline components/cli/pkg/commands/setup_switch.go]: Shall we move this into kubectl package?

File: components/cli/cmd/cellery/setup_cleanup_gcp.go (hunk 6 above)
Comment (MadhukaHarith92) [reply components/cli/cmd/cellery/setup_cleanup_gcp.go]: updated

File: components/cli/cmd/cellery/setup_cleanup_gcp.go (hunk 5 above)
Comment (MadhukaHarith92) [reply components/cli/cmd/cellery/setup_cleanup_gcp.go]: moved the validation to Run function

File: components/cli/cmd/cellery/setup_cleanup_gcp.go (hunk 4 above)
Comment (MadhukaHarith92) [reply components/cli/cmd/cellery/setup_cleanup_gcp.go]: Ran make code.format.

File: components/cli/cmd/cellery/setup_cleanup_local.go (hunk 7 above)
Comment (MadhukaHarith92) [reply components/cli/cmd/cellery/setup_cleanup_local.go]: Updated.

File: components/cli/cmd/cellery/setup_create_existing_cluster.go (hunk 8 above)
Comment (MadhukaHarith92) [reply components/cli/cmd/cellery/setup_create_existing_cluster.go]: updated.

File: components/cli/pkg/commands/setup_switch.go (hunk 9 above)
Comment (MadhukaHarith92) [reply components/cli/pkg/commands/setup_switch.go]: Moved.

File: components/cli/cmd/cellery/setup_cleanup_existing.go (hunk 3 above)
Comment (MadhukaHarith92) [reply components/cli/cmd/cellery/setup_cleanup_existing.go]: Updated.
